website/tfe.erb

```diff
@@ -0,0 +1,66 @@
+<% wrap_layout :inner do %>
+    <% content_for :sidebar do %>
+        <div class="docs-sidebar hidden-print affix-top" role="complementary">
+            <ul class="nav docs-sidenav">
+                <li<%= sidebar_current("docs-home") %>>
+                    <a href="/docs/providers/index.html">All Providers</a>
+                </li>
+
+                <li<%= sidebar_current("docs-tfe-index") %>>
+                    <a href="/docs/providers/tfe/index.html">Terraform Enterprise Provider</a>
+                </li>
+
+                <li<%= sidebar_current("docs-tfe-resource") %>>
+                    <a href="#">Resources</a>
+                    <ul class="nav nav-visible">
+                        <li<%= sidebar_current("docs-tfe-organization") %>>
```

Comment:
When visiting the `tfe_organization` page it will highlight both `tfe_organization` and `tfe_organization_token` since the `sidebar_current` entry prefixes the other resource.

To workaround this with other providers, we append `-x` to the end of problematic cases like this, e.g. `docs-tfe-organization-x` here in the ERB and in the Markdown topmatter.

As another nitpick: We generally use `docs-resource-` instead of `docs-` in preparation for supporting data sources in the future (without needing to rewrite these later).

Reply:
Good points! Fixed in next commit.